google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfigurator.java

```diff
@@ -112,9 +69,19 @@ public AppEngineDeploymentConfiguration createDefaultConfiguration(
   public SettingsEditor<AppEngineDeploymentConfiguration> createEditor(
       @NotNull DeploymentSource source,
       @NotNull RemoteServer<AppEngineServerConfiguration> server) {
+    if (!(source instanceof AppEngineDeployable)) {
```

Comment:
No, it can't happen currently based on the logic that collects the deployment sources. I was just being defensive.